core/tee/tee_svc.c

```diff
@@ -43,6 +42,12 @@
 #include <trace.h>
 #include <kernel/trace_ta.h>
 #include <kernel/chip_services.h>
+#include <kernel/misc.h>
+
+#define MAX_BASE64_BUFFER_SIZE 120
+
+extern int base64_encode(const unsigned char *in,  unsigned long len,
```

Comment:
Libtomcrypt functions cannot be called directly in the code as libtomcrypt may be replaced by another library. The abstraction layer must be used instead.